crates/turbo-tasks-fs/src/rope.rs

```diff
@@ -438,18 +438,18 @@ impl From<Vec<u8>> for Uncommitted {
 
 impl InnerRope {
     /// Returns a String instance of all bytes.
-    pub fn to_str(&self) -> Result<Cow<'_, str>> {
+    fn to_str(&self, len: usize) -> Result<Cow<'_, str>> {
         match &self[..] {
             [] => Ok(Cow::Borrowed("")),
-            [Shared(inner)] => inner.to_str(),
+            [Shared(inner)] => inner.to_str(len),
             [Local(bytes)] => {
                 let utf8 = std::str::from_utf8(bytes);
                 utf8.context("failed to convert rope into string")
                     .map(Cow::Borrowed)
             }
             _ => {
                 let mut read = RopeReader::new(self, 0);
-                let mut string = String::with_capacity(self.len());
```

Comment:
I can't believe I mixed up the `InnerRope`'s vec length with the `Rope`'s contents length. 🤦

Reply:
I doesn't matter that much anyway^^